overseer.go

```diff
@@ -59,6 +59,8 @@ type Config struct {
 	NoRestartAfterFetch bool
 	//Fetcher will be used to fetch binaries.
 	Fetcher fetcher.Interface
+	//Supervise the child process (ensure it stays running)
```

Comment:
nit: space after `//` looks like your editor reformatted all the other comments and added the space but missed this one lol.